### ISSUE TITLE
Uncaught error when using mirrorsharp

### ISSUE THREAD

**Issue by Sicos1977** [ashmind/mirrorsharp#182]
When using MirrorSharp (that is running fine) I see this error when looking in the dev console from Chrome. Someone that has an idea where this is comming from and how to fix it?

```
Uncaught Error: System.TypeInitializationException: The type initializer for 'MirrorSharp.Internal.Reflection.SignatureHelpProviderWrapper' threw an exception. ---> System.ArgumentException: Incorrect number of arguments supplied for call to method 'System.Threading.Tasks.Task`1[Microsoft.CodeAnalysis.SignatureHelp.SignatureHelpItems] GetItemsAsync(Microsoft.CodeAnalysis.Document, Int32, Microsoft.CodeAnalysis.SignatureHelp.SignatureHelpTriggerInfo, Microsoft.CodeAnalysis.SignatureHelp.SignatureHelpOptions, System.Threading.CancellationToken)'
   at System.Linq.Expressions.Expression.ValidateArgumentCount(MethodBase method, ExpressionType nodeKind, Int32 count, ParameterInfo[] pis)
   at System.Linq.Expressions.Expression.ValidateArgumentTypes(MethodBase method, ExpressionType nodeKind, ReadOnlyCollection`1& arguments)
   at System.Linq.Expressions.Expression.Call(Expression instance, MethodInfo method, IEnumerable`1 arguments)
   at MirrorSharp.Internal.Reflection.SignatureHelpProviderWrapper.CompileGetItemsAsyncWithUntypedResult()
   at MirrorSharp.Internal.Reflection.SignatureHelpProviderWrapper..cctor()
   --- End of inner exception stack trace ---
   at MirrorSharp.Internal.Reflection.SignatureHelpProviderWrapper.IsTriggerCharacter(Char ch)
   at MirrorSharp.Internal.Handlers.Shared.SignatureHelpSupport.<TryApplySignatureHelpAsync>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at MirrorSharp.Internal.Handlers.Shared.SignatureHelpSupport.<TryApplySignatureHelpAsync>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MirrorSharp.Internal.Handlers.Shared.TypedCharEffects.<ApplyTypedCharAsync>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MirrorSharp.Internal.Connection.<ReceiveAndProcessInternalAsync>d__14.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MirrorSharp.Internal.Connection.<ReceiveAndProcessAsync>d__13.MoveNext()
    at Object.serverError (http://localhost:5875/scripts/mirrorsharp/main/editor.js:438:15)
    at Editor.<computed> (http://localhost:5875/scripts/mirrorsharp/main/editor.js:154:36)
    at WebSocket.<anonymous> (http://localhost:5875/scripts/mirrorsharp/main/connection.js:66:13)
```

**Comment by ashmind** [comment]
Thanks for reporting! This was a relatively large issue with specific version of Roslyn -- I plan for it to be fixed in MirrorSharp.Common 3.0.0 (currently available as [3.0.0-test-2022-04-02-1](https://www.nuget.org/packages/MirrorSharp.Common/3.0.0-test-2022-04-02-1)).

**Comment by ashmind** [comment]
This should no longer happen in the latest version, please create a new issue if it reappears (unlikely to be the same problem).
